src/Microsoft.AspNetCore.Razor.Language/ITagHelperDescriptorProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public interface ITagHelperDescriptorProvider : IRazorEngineFeature
```

Comment:
Yeah we'll need to delete the one from `.Legacy`. I went ahead and did that. We'll need to sync this change with tooling because of this.